payment/src/main/java/org/killbill/billing/payment/core/sm/control/CompletionControlOperation.java

```diff
@@ -107,13 +111,12 @@ public OperationResult doOperationCallback() throws OperationException {
                     } else {
                         throw new OperationException(null, executePluginOnFailureCallsAndSetRetryDate(updatedPaymentControlContext));
                     }
-                } catch (final PaymentApiException e) {
+                } catch (final PaymentApiException | RuntimeException e) {
```

Comment:
We actually don't use Multi-Catch statements -- could you revert this?

Reply:
Yes of course... Sorry, Idea inspections :sweat_smile:
It is fine to add a commit here, or you prefer me to close this, force-push an amended commit, and make new PR?

Reply:
Nevermind, I force-pushed. I hope to not have break anything... :smile: